gradle/libs.versions.toml

```diff
@@ -159,6 +161,7 @@ jaxb-runtime = { module = "org.glassfish.jaxb:jaxb-runtime", version.ref = "jaxb
 kafka-clients = { module = "org.apache.kafka:kafka-clients", version.ref = "kafka" }
 kafka-connect-api = { module = "org.apache.kafka:connect-api", version.ref = "kafka" }
 kafka-connect-json = { module = "org.apache.kafka:connect-json", version.ref = "kafka" }
+kafka-connect-transforms = { module = "org.apache.kafka:connect-transforms", version.ref = "kafka" }
```

Comment:
We need to verify this addition is captured in LICENSE/NOTICE files (if separate from the other kafka dependencies)

Reply:
added section for kafka libs to LICENSE

Reply:
You can remove this too for now, until we add the transforms to the runtime.

Reply:
@bryanck do you mean remove all grade setting/changes and add it with separate PR?

Reply:
I meant skip any of the runtime license changes, until we add the transforms lib to the runtime

Reply:
in that case i have reverted all the changes done to LICENSE/NOTICE files.